README.md

```diff
@@ -13,7 +15,7 @@ Example Usage
 
 ```hcl
 module "foo" {
-  source = "git@github.com:cites-illinois/as-aws-modules//ecr"
+  source = "git@github.com:techservicesillinois/terraform-aws-ecr"
 
   name = "repoName"
   writers = ["arn:aws:iam::874445906176:root"]
```

Comment:
Below in `disable_lifecycle_policy` the word 'true' should be `true` i.e. the markdown is wrong.